apps/ria-e2e/cypress.config.ts

```diff
@@ -1,6 +1,15 @@
 import { defineConfig } from 'cypress';
-import { nxE2EPreset } from '@nrwl/cypress/plugins/cypress-preset';
+import { nxE2EPreset } from '@nx/cypress/plugins/cypress-preset';
 
 export default defineConfig({
-  e2e: nxE2EPreset(__filename),
+  e2e: {
+    ...nxE2EPreset(__filename),
+    /**
+     * TODO(@nx/cypress): In Cypress v12,the testIsolation option is turned on by default.
```

Comment:
should I create an issue for this?

Reply:
Should be fine as long as this option is not removed by Cypress in this major version.